README.md

```diff
@@ -41,6 +41,7 @@ In your project, go to the debugger and hit the little gear icon and choose _PHP
 #### Supported launch.json settings:
 
 - `request`: Always `"launch"`
+- `bind`: The address to bind to when listening for XDebug (default: all IPv6 connections if available, else all IPv4 connections)
```

Comment:
I find this name a bit confusing. The argument to `server.listen()` is named `hostname`. Could we name it that?

Reply:
We can, I kind of find hostname confusing too in this context - it sounds like the name of the host that XDebug is connecting from. I've seen it named `bind` in other applications so that's why I used that name here. I think `hostname` makes sense in the context of a call to `server.listen()` but in the context of a configuration for this module it's ambiguous.